openerp/tools/translate.py

```diff
@@ -742,18 +742,20 @@ def get_root_view(xml_id):
                 _logger.error("name error in %s: %s", xml_name, str(exc))
                 continue
             objmodel = registry.get(obj.model)
-            if (objmodel is None or field_name not in objmodel._columns
+            if (objmodel is None or (
+                        field_name not in objmodel._columns and
```

Comment:
Please, allow me to understand: are there any objmodel._columns which are not part of  objmodel._fields? If so, could you please provide an example?